packages/jobs/src/adapters/PrismaAdapter.ts

```diff
@@ -232,6 +234,7 @@ export class PrismaAdapter extends BaseAdapter {
   }
 
   async clear() {
+    this.options.db.$disconnect()
```

Comment:
Why disconnect right *before* making the database call? Wouldn't the next line just cause a connection to immediately re-establish?

Also you can use just `this.db` here (the important props are extracted from `options` and set right on `this`).

Reply:
oops, that wasn't ment to be in there

Reply:
That's from me testing the types....